fossology/fossology
# feat(bulk): inclusion of licensetext, acknowledgement and comment

**HOW TO TEST**

- Upload any package and go to single file view.
- Click on button "bulk recognition" add license with acknowledgement, license text and comment.
- Schedule the bulk scan and check if the added acknowledgement, license text and comment to the decided file.
